addons/website_slides/static/src/scss/slide_views.scss

```
@@ -1,12 +1,8 @@
 $o-kanban-large-record-width: 400px;
 
 .o_kanban_view .o_slide_kanban {
-    .o_kanban_group:not(.o_column_folded) {
-        width: $o-kanban-large-record-width + 2*$o-kanban-group-padding;
-    }
-    .o_kanban_record {
-        width: $o-kanban-large-record-width;
-    }
+    --KanbanRecord-width: #{$o-kanban-large-record-width};
```

For the master port, should we keep defining a variable if it's only used in one place ?
```suggestion
    --KanbanRecord-width: 400px;
```